qcengine/config.py

```diff
@@ -14,8 +14,11 @@
 import pydantic
 import yaml
 
+from ._version import get_versions
```

Comment:
On this I noted that repeated `get_versions()` is fairly slow. Can we add this once to GLOBALS and then pull it from there everywhere else?